.github/workflows/ci.yaml

```diff
@@ -50,6 +50,10 @@ jobs:
         run: |
           pytest tests/integration/ -v --tb=short
 
+      - name: Type check with mypy
+        run: |
+          mypy gnucash_uk_vat/
```

Comment:
If you specify the file paths in the config, you won't need to pass any arguments here. See, for example:
https://github.com/aio-libs/aiohttp/blob/eac0ef9291fe34912809619086fe4e6c859e2cc7/.mypy.ini#L2

(you may also want to enable strict mode or a bunch of other options in that config, to help find more errors).